source/android/adaptivecards/src/AdaptiveCardObjectModel.i

```diff
@@ -139,7 +141,6 @@ struct tm {
 %shared_ptr(AdaptiveCards::SubmitActionParser)
 %shared_ptr(AdaptiveCards::ImageSetParser)
 %shared_ptr(AdaptiveCards::DateInputParser)
-%shared_ptr(AdaptiveCards::DateTimePreparsedToken)
```

Comment:
>) [](start = 49, length = 1)

Why are we removing this?

Reply:
I dont even know how did this happen, reverting this deletion

---
In reply to: [207609313](https://github.com/Microsoft/AdaptiveCards/pull/1759#discussion_r207609313) [](ancestors = 207609313)